mapswipe_workers/mapswipe_workers/mapswipe_workers.py

```diff
@@ -41,6 +42,7 @@ def type_cast_value(self, ctx, value):
 @click.version_option()
 def cli(verbose):
     """Enable logging."""
+    create_directories()
```

Comment:
Hey,
maybe you could briefly explain to me again, why you think that it's better to create the directories here and not during the initialization of the docker container.